crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -239,6 +267,7 @@ struct Order {
     kind: Kind,
     partially_fillable: bool,
     class: Class,
+    protocol_fees: Vec<FeePolicy>,
```

Comment:
`fee_policies` seems more fitting since technically we could have orders that only have a partner fee which the protocol will never get, right?